src/resources/user/index.js

```diff
@@ -1,9 +1,11 @@
 const Router = require('koa-router');
+const validate = require('middlewares/validate');
+const validators = require('./validators');
 
 const router = new Router();
 const controller = require('./user.controller');
 
 router.get('/current', controller.getCurrent);
-router.put('/current', controller.updateCurrent);
+router.put('/current', validate(validators.update), controller.updateCurrent);
```

Comment:
Looks really great but I think that we also must add ability to start validator from controller because sometimes we want start validation after some actions.

Example:
We want to update information of the client:
1) Check that client exists
2) Check that we have permissions to update information of this client
3) Start validation of the data
4) Do some another actions